build.gradle

```diff
@@ -33,6 +33,140 @@ subprojects {
                     "--add-opens", "java.management/sun.management=ALL-UNNAMED",
                     "--add-opens", "jdk.management/com.sun.management.internal=ALL-UNNAMED"]
     }
+
+    jar {
+        metaInf {
+            from("$projectDir/../../LICENSE")
+        }
+
+        afterEvaluate {
+            manifest {
+
+                var description = Objects.nonNull(project.description) ? project.description : 'No description'
+                var specTitle = project.extensions.extraProperties.has('prettyName')
+                        ? project.extensions.extraProperties.get('prettyName')
+                        : 'Telestion Module'
+
+                attributes(
+                        'Build-Timestamp': new java.text.SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ").format(new Date()),
+                        'Created-By': "Gradle ${gradle.gradleVersion}",
+                        'Build-Jdk': "${System.properties['java.version']} (${System.properties['java.vendor']} ${System.properties['java.vm.version']})",
+                        'Build-Jdk-Spec': "${System.properties['java.version']}",
+                        'Build-OS': "${System.properties['os.name']} ${System.properties['os.arch']} ${System.properties['os.version']}",
+                        'Specification-Title': specTitle,
+                        'Specification-Version': project.version,
+                        'Specification-Vendor': 'WueSpace e.V.',
+                        'Implementation-Title': project.name,
+                        'Implementation-Version': project.version,
+                        'Implementation-Vendor': 'de.wuespace.telestion',
+                        'X-Compile-Target-JDK': gradle.ext.targetCompatibility,
+                        'X-Compile-Source-JDK': gradle.ext.sourceCompatibility,
+                        'Description': description
+                )
+            }
+        }
+    }
+
+    publishing {
+        afterEvaluate {
+            publications {
+                maven(MavenPublication) {
+                    from components.java
+
+                    pom {
+                        name = project.name
+                        description = project.description
+                        url = 'https://telestion.wuespace.de/'
+
+                        licenses {
+                            license {
+                                name = 'MIT License'
+                                url = 'https://opensource.org/licenses/mit-license.php'
+                            }
+                        }
+
+                        developers {
```

Comment:
Are these added manually or do we have an action (already) which automatically puts them in?

Reply:
Nope, it's hard coded.
Some time in the future we maybe have external contributers, so I think it's not so useful to automate this step :thinking: